metagraph/src/cli/config/config.cpp

```diff
@@ -218,6 +218,8 @@ Config::Config(int argc, char *argv[]) {
             align_one_strand = true;
         } else if (!strcmp(argv[i], "--align-edit-distance")) {
             alignment_edit_distance = true;
+        } else if (!strcmp(argv[i], "--align-chain")) {
+            alignment_chain = true;
```

Comment:
Add a line to the usage description? Otherwise, no one will know that this parameter exists.